app/src/main/java/org/p2p/wallet/utils/StdExtensions.kt

```diff
@@ -0,0 +1,5 @@
+package org.p2p.wallet.utils
+
+fun Result<*>.invokeAndForget() {
```

Comment:
maybe we will discuss this kind of sugar functions tomorrow , for now i don't see reasons to have them